### PR TITLE
Split RGB Matrix Zygomorph support

### PR DESCRIPTION
**Pull request by XScorpion2**
## Description

This is a change to the RGBKB Zygomorph by @Legonut that started me down the crazy idea of implementing Split Keyboard RGB Matrix support.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by Legonut** [comment]
Approved for Zygomorph development

**Comment by XScorpion2** [comment]
Not stale, just moving and trying to find shit

**Comment by tzarc** [comment]
Sorry about that, GitHub decided to delete the `develop` branch from upstream when we merged it, despite being told otherwise. Reopened.

**Comment by drashna** [comment]
Needs a rebase.

**Comment by XScorpion2** [comment]
> 
> 
> Needs a rebase.

rebase was done.

File: keyboards/rgbkb/zygomorph/keymaps/xulkal/rules.mk [hunk 1]
```diff
@@ -8,11 +8,13 @@ EXTRAKEY_ENABLE = yes       # Audio control and System control
...
-RGBLIGHT_ANIMATIONS = yes   # LED animations
-RGBLIGHT_SPLIT_ENABLE = yes # Split RGBLight Support
-RGB_MATRIX_ENABLE = no      # Enable per-key coordinate based RGB effects. Do not enable with RGBlight
+RGBLIGHT_ENABLE = no        # Enable global lighting effects. Do not enable with RGB Matrix
+RGBLIGHT_ANIMATIONS = no    # LED animations
+RGBLIGHT_SPLIT_ENABLE = no  # Split RGBLight Support
+RGB_MATRIX_ENABLE = yes  	# Enable per-key coordinate based RGB effects. Do not enable with RGBlight
+RGB_MATRIX_DRIVER = WS2812
 RGB_MATRIX_KEYPRESSES = no  # Enable reactive per-key effects.
+SPLIT_TRANSPORT = mirror    # For split RGB Matrix support
```
Comment (drashna) [inline keyboards/rgbkb/zygomorph/keymaps/xulkal/rules.mk:17]:
```suggestion
```

File: keyboards/rgbkb/zygomorph/rev1/config.h [hunk 2]
```diff
@@ -55,7 +55,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     #define RGBLED_NUM 60
     #define RGBLED_SPLIT { 30, 30 }
 #endif
-#define DRIVER_LED_TOTAL 30
+
+#define DRIVER_LED_TOTAL 60
+#define RGB_MATRIX_SPLIT { 30, 30 }
```
Comment (drashna) [inline keyboards/rgbkb/zygomorph/rev1/config.h:60]:
```suggestion
#define RGB_MATRIX_SPLIT { 30, 30 }
#define SPLIT_TRANSPORT_MIRROR
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/rgbkb/zygomorph/rev1/config.h (hunk 2 above)
Comment (XScorpion2) [reply keyboards/rgbkb/zygomorph/rev1/config.h:60]: The split rgb matrix work is only enabled for my keymap, as the base zygomorph build instructions don't support split rgb matrix as you have to desolder a jumper.

File: keyboards/rgbkb/zygomorph/keymaps/xulkal/rules.mk (hunk 1 above)
Comment (XScorpion2) [reply keyboards/rgbkb/zygomorph/keymaps/xulkal/rules.mk]: fixed

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**